artifacthub/library/general/volumeresources/1.0.0/template.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: k8svolumerequests
```

Comment:
since this policy validates specifically for emptydir volume size limit, the name, description should be updated to reflect this.

Reply:
@ritazh  change name to `k8semptydirvolumesizelimit`?